dangerzone/isolation_provider/base.py

```diff
@@ -72,10 +73,19 @@ def convert(
             conversion_proc = self.start_doc_to_pixels_proc()
             with tempfile.TemporaryDirectory() as t:
                 Path(f"{t}/pixels").mkdir()
-                self.doc_to_pixels(document, t, conversion_proc)
-                conversion_proc.wait(3)
-                # TODO: validate convert to pixels output
```

Comment:
The TODO existed there since c9c01f6e79ea6749216f705e65b65530feeea3c0. That's back when the pixels where stored as individual files. Now that the pixels are not stored as files, but are read from stdout, and have the proper size, there's not much else to validate. It's up to PyMuPDF to handle them properly.